session-signatures/getLitActionSessionSigs/src/index.ts

```diff
@@ -48,20 +50,40 @@ export const getSessionSigsLitAction = async () => {
       `✅ Minted new PKP with public key: ${pkp.publicKey} and ETH address: ${pkp.ethAddress}`
     );
 
+    console.log("🔄 Minting Capacity Credits NFT...");
+    const capacityTokenId = (
+      await litContracts.mintCapacityCreditsNFT({
+        requestsPerKilosecond: 10,
+        daysUntilUTCMidnightExpiration: 1,
+      })
+    ).capacityTokenIdStr;
+    console.log(`✅ Minted new Capacity Credit with ID: ${capacityTokenId}`);
+
+    console.log("🔄 Creating capacityDelegationAuthSig...");
+    const { capacityDelegationAuthSig } =
+      await litNodeClient.createCapacityDelegationAuthSig({
+        dAppOwnerWallet: ethersSigner,
+        capacityTokenId,
+        delegateeAddresses: [pkp.ethAddress],
+        uses: "1",
+      });
+    console.log(`✅ Created the capacityDelegationAuthSig`);
+
     console.log("🔄 Adding example permitted Lit Action to the PKP");
+    const litActionCode = `(async () => {LitActions.setResponse({ response: makeItTrue });})();`;
```

Comment:
If we want to showcase passing a param to the Lit Action code, then I think we should do something different:

```
(() => {
    if (magicNumber >= 42) {
        LitActions.setResponse({ response:"true" });
    } else {
        LitActions.setResponse({ response: "false" });
    }
})();

Reply:
Changed this. Just wanted to confirm, even when the magicNumber is less than 42 should sessionsigs still be generated? (they are)

Reply:
No, they shouldn't, not sure why they are

Reply:
Works!